README.md

```diff
@@ -70,11 +70,15 @@ Use Android Studio's Preferences to install NDK (Side by side):
 
```

Comment:
^^^ For installing CocoaPods we should use ```brew install cocoapods```. The user will also need to run ```pod install``` prior to trying to run the iOS app.

Reply:
`pod install` is automatically run when the user runs `yarn`. Don't think we need to have that added to the read me